azurelinuxagent/common/protocol/extensions_goal_state_from_extensions_config.py

```diff
@@ -24,14 +24,14 @@
 from azurelinuxagent.common.exception import ExtensionsConfigError
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.protocol.extensions_goal_state import ExtensionsGoalState
-from azurelinuxagent.common.protocol.restapi import Extension, ExtHandler, ExtHandlerVersionUri, VMAgentManifest, \
-    VMAgentManifestUri, ExtensionState, ExtHandlerList, VMAgentManifestList, InVMGoalStateMetaData
+from azurelinuxagent.common.protocol.restapi import Extension, ExtHandler, VMAgentManifest, \
+    ExtensionState, ExtHandlerList, InVMGoalStateMetaData
 from azurelinuxagent.common.utils.textutil import parse_doc, parse_json, findall, find, findtext, getattrib, gettext, format_exception, \
     is_str_none_or_whitespace, is_str_empty
 
 
 class ExtensionsGoalStateFromExtensionsConfig(ExtensionsGoalState):
-    def __init__(self, incarnation, xml_text, wire_client=None):
```

Comment:
The default values (None) for wire_client was a left over of a test I was doing during my previous PR.